source/gsoc-admin-tasks.md

```diff
@@ -22,21 +22,21 @@ contributors to your projects.
 If you want to participate with your project you now need to
 
  - understand what it means to be a mentor [1]
- - propose your project ideas. Just label your issues with gsoc2011 in JIRA and
+ - propose your project ideas. Just label your issues with gsoc20XX in JIRA and
```

Comment:
could you please use `2021` instead of `20XX`
I'll try to do better search next time and replace all occurrences :))